.github/workflows/hyperspace-docker-image.yml

```diff
@@ -3,16 +3,27 @@ name: "Build and publish Hyperspace Docker image"
 on:
   push:
     branches:
-      - master
+      - '*'
```

Comment:
we'll be pushing on all branches now?

Reply:
I thought the same. But needs to have a tag with v as a prefix